boards/airfy-beacon/doc.txt

```diff
@@ -0,0 +1,209 @@
+/**
+ * @defgroup    boards_airfy-beacon Airfy Beacon
+ * @ingroup     boards
+ * @brief       Support for the Airfy Beacon board
+ *
+ * ## Overview
+ * 
+ * The Airfy Beacon is utilizing a Nordics NRF51822QFAA SoC.
+ * The SoC features 16Kb of RAM, 256Kb of flash ROM and comes on top of the usual micro-controller peripherals with a 2.4GHz radio that supports both Nordics proprietary ShockBurst as well as Bluetooth Low Energy (BLE).
+ * 
+ * The board was available via [Indiegogo](https://www.indiegogo.com/projects/airfy-beacon-make-your-smart-home-even-smarter).
+ * 
+ * ## Hardware
+ * 
+ * ![airfy-beacon](https://raw.githubusercontent.com/wiki/RIOT-OS/RIOT/images/airfy-beacon.jpg)
+ * 
+ * | MCU 		| NRF51822QFAA 		|

```

Comment:
should use spaces instead of tabs...

Reply:
The original file in the wiki already contains the tabs, maybe the automatic script could also handle that. What do you think @jia200x ?

Reply:
Bash magic may do the trick :)

Reply:
`expand` may help here